Apps/Sandcastle/gallery/Multiple Synced Views.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="Multiple views synced across time and space.">
+    <meta name="cesium-sandcastle-labels" content="Beginner, Showcases">
+    <title>Cesium Demo</title>
+    <script type="text/javascript" src="../Sandcastle-header.js"></script>
+    <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.20/require.js"></script>
+    <script type="text/javascript">
+    require.config({
+        baseUrl : '../../../Source',
+        waitSeconds : 60
+    });
+    </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<style>
+    @import url(../templates/bucket.css);
+    #cesiumContainer {
+      display: flex;
+      width: 100%;
+      height: 100%;
+    }
+    #view3D {
+      display: inline-block;
+      width: 100%;
+    }
+    #view2D {
+      display: inline-block;
+      width: 100%;
+    }
+</style>
+<div id="cesiumContainer" class="fullSize">
+    <div id="view3D"></div>
+    <div id="view2D"></div>
+</div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar"></div>
+<script id="cesium_sandcastle_script">
+function startup(Cesium) {
+    'use strict';
+//Sandcastle_Begin
+// We want our two views to be synced across time, so we create
+// a shared clock object that both views share
+var clockViewModel = new Cesium.ClockViewModel(new Cesium.Clock());
```

Comment:
The `clock` parameter is optional -  https://cesiumjs.org/Cesium/Build/Documentation/ClockViewModel.html

Since we're not setting any specific configurations, I would omit it.